Zend/zend_execute.c

```diff
@@ -2998,6 +2998,47 @@ ZEND_API void ZEND_FASTCALL zend_check_missing_arg(zend_execute_data *execute_da
 	zend_verify_missing_arg(execute_data, arg_num, cache_slot);
 }
 
+static inline zval* zend_anon_fetch_prop(zval *object, zend_string *prop, int type, zval *val) {
+	zval name;
+	zend_class_entry *scope = EG(scope);
+	zval *val_ptr;
+	zend_string *key;
+	ZVAL_STR(&name, prop);
+
+	EG(scope) = Z_OBJCE_P(object);
+
+	val_ptr = Z_OBJ_HT_P(object)->get_property_ptr_ptr(object, &name, type, NULL);
```

Comment:
I'm pretty sure that this can never return anything other than a valid value now, since properties are always declared.